src/cache.rs

```diff
@@ -403,6 +397,20 @@ impl Caches {
         })
     }
 
+    /// Clear the temporary files.
```

Comment:
I think we can make a case for emitting an error to Sentry if the tmpdir is not empty on restart

Reply:
This should only happen after a crash and there probably have been alerts for this already in that case?  I fear this might just lead to duplicate reporting.  It's not really a bad failure mode, just a backstop to some files never being cleaned up